examples/lfric/eg14/acc_parallel_dm.py

```diff
@@ -64,4 +66,7 @@ def trans(psy):
             parallel_trans.apply(loop.parent.parent)
         enter_data_trans.apply(schedule)
 
+        for kernel in schedule.walk(CodedKern):
```

Comment:
Add a comment here that makes it explicit that the kernel is modified (or a new kernel file is created).

Reply:
Oh yes. In fact the various doc strings were outdated too.